jest.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  collectCoverageFrom: ['src/**/*.{js,jsx,mjs}', 'backend/**/*.{js,jsx,mjs}'],
+  setupFiles: ['<rootDir>/config/polyfills.js'],
+  setupTestFrameworkScriptFile: '<rootDir>/config/jest/setupTests.js',
+  testMatch: [
+    '<rootDir>/src/**/__tests__/**/*.{js,jsx,mjs}',
+    '<rootDir>/(src|backend)/**/?(*.)(spec|test).{js,jsx,mjs}',
+  ],
+  testEnvironment: 'jsdom',
```

Comment:
this line was the change, not sure if we want our jest config to be in package.json or here, happy to move back

Reply:
Fine by me 👍